source/MapDetailPanel.cpp

```diff
@@ -586,18 +588,19 @@ void MapDetailPanel::DrawKey()
 	else if(commodity == SHOW_GOVERNMENT)
 	{
 		// Each system is colored by the government of the system. Only the
-		// four largest visible governments are labeled in the legend.
-		vector<pair<double, const Government *>> distances;
-		for(const auto &it : closeGovernments)
+		// `availableLines` largest visible governments are labeled in the legend, as
+		// well as uninhabited & unexplored (if applicable).
+		vector<pair<unsigned int, const Government *>> screenNum;
```

Comment:
```suggestion
		vector<pair<const Government *, unsigned int>> screenNum;
```
so it is inline with governmentCounts